sdk/typescript/graphql/client.ts

```diff
@@ -14,6 +35,7 @@ class CustomSetter {
 
 export function createGQLClient(port: number, token: string): GraphQLClient {
   const client = new GraphQLClient(`http://127.0.0.1:${port}/query`, {
+    fetch: createFetchWithTimeout(1000 * 60 * 60 * 24 * 7), // 1 week timeout so we should never hit that one.
```

Comment:
nit: probably worth commenting the logic for why we have such a high timeout.

Reply:
Done